config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -3667,10 +3667,6 @@ spec:
                       performed
                     format: date-time
                     type: string
-                  lastProcessModuleConfigUpdate:
```

Comment:
you need rebase :)

Reply:
github seems to think its fine :D

Reply:
<img width="832" alt="image" src="https://github.com/Dynatrace/dynatrace-operator/assets/1374633/cb0f2642-dc48-47a4-8905-050da5a90150">